wp-content/plugins/jetpack/_inc/build/comment-likes/comment-like-count.min.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array(), 'version' => '53667c0d8e2fcf69f3a0f0f4cdbe62d1');
```

Comment:
:no_entry_sign: **Error( severity 5 )**: When a multi-item array uses associative keys, each value should start on a new line (*WordPress.Arrays.ArrayDeclarationSpacing.AssociativeArrayFound*).